articles/how-to-fix-and-prevent-sql-injection-in-wordpress/index.md

```diff
@@ -0,0 +1,196 @@
+### Introduction
+SQL injection in [WordPress](https://kinsta.com/knowledgebase/what-is-wordpress/) is [ranked the second most critical vunerability](secure.wphackedhelp.com). [Updating your WordPress website](https://wordpress.org/support/article/updating-wordpress/) to the latest release shields you from SQL injection. Nevertheless, third party themes and plugins still pose a big threat to WordPress users. When you hear SQL injection, you understand it is the manipulation of a security flaw in database systems. WordPress as a platform runs server-side scripts in PHP. SQl and PHP hence make WordPress vulnerable to attackers via inserting malicious URLs embedded with commands that trigger behaviours from the database. Consequently,  sensitive information from your database is exposed or leaked to hackers which they can use to manipulate your WordPress website.
```

Comment:
"Hence, SQL and PHP make WordPress" would flow better. 

To aid readability: Add a full stop after attackers are rework the sentence like so: "attackers. They could insert malicious URLs (which are embedded with commands) that trigger unwanted behaviours from your database."